packages/platform/atoms/event-types/hooks/useTabsNavigations.tsx

```diff
@@ -0,0 +1,220 @@
+      });
+    }
+    navigation.splice(1, 0, {
+      name: "availability",
+      href: `/event-types/${formMethods.getValues("id")}?tabName=availability`,
+      icon: "calendar",
+      info:
+        isManagedEventType || isChildrenManagedEventType
+          ? formMethods.getValues("schedule") === null
+            ? "members_default_schedule"
+            : isChildrenManagedEventType
+            ? `${
+                formMethods.getValues("scheduleName")
+                  ? `${formMethods.getValues("scheduleName")} - ${t("managed")}`
+                  : `default_schedule_name`
+              }`
+            : formMethods.getValues("scheduleName") ?? `default_schedule_name`
+          : formMethods.getValues("scheduleName") ?? `default_schedule_name`,
+    });
+    // If there is a team put this navigation item within the tabs
+    if (team) {
+      navigation.splice(2, 0, {
+        name: "assignment",
+        href: `/event-types/${formMethods.getValues("id")}?tabName=team`,
+        icon: "users",
+        info: `${t(watchSchedulingType?.toLowerCase() ?? "")}${
+          isManagedEventType ? ` - ${t("number_member", { count: watchChildrenCount || 0 })}` : ""
+        }`,
+      });
+    }
+    const showWebhooks = !(isManagedEventType || isChildrenManagedEventType);
+    if (showWebhooks) {
+      if (team) {
+        navigation.push({
+          name: "instant_tab_title",
+          href: `/event-types/${eventType.id}?tabName=instant`,
+          icon: "phone-call",
+          info: `instant_event_tab_description`,
+        });
+      }
+      navigation.push({
+        name: "webhooks",
+        href: `/event-types/${formMethods.getValues("id")}?tabName=webhooks`,
+        icon: "webhook",
+        info: `${activeWebhooksNumber} ${t("active")}`,
+      });
+    }
+    const hidden = true; // hidden while in alpha trial. you can access it with tabName=ai
+    if (team && hidden) {
+      navigation.push({
+        name: "Cal.ai",
+        href: `/event-types/${eventType.id}?tabName=ai`,
+        icon: "sparkles",
+        info: "cal_ai_event_tab_description", // todo `cal_ai_event_tab_description`,
+      });
+    }
+    return navigation;
+  }, [
+    t,
+    enabledAppsNumber,
+    installedAppsNumber,
+    enabledWorkflowsNumber,
+    availability,
+    isManagedEventType,
+    isChildrenManagedEventType,
+    team,
+    length,
+    requirePayment,
+    multipleDuration,
+    formMethods.getValues("id"),
+    watchSchedulingType,
+    watchChildrenCount,
+    activeWebhooksNumber,
+  ]);
+
+  return { tabsNavigation: EventTypeTabs };
+};
+
+type getNavigationProps = {
+  t: TFunction;
+  length: number;
+  id: number;
+  multipleDuration?: EventTypeSetupProps["eventType"]["metadata"]["multipleDuration"];
+  enabledAppsNumber: number;
+  enabledWorkflowsNumber: number;
+  installedAppsNumber: number;
+  availability: AvailabilityOption | undefined;
+};
+
+function getNavigation({
```

Comment:
This hook seems very similar to platform tabs navigation - i assume the difference is that web app has apps tab and platform has payments tab and each could be toggled bassed on useIsPlatform within the use tabs hook itself. Then i see "href" for each navigation differs too, but in web app we can also just use `window.location.href` instead of having relative href links.

I am thinking can web and platform tab hooks can be combined into one, and we do not pass platform tabs as prop but just within hook itself useIsPlatform and then based on that show / hide tabs.

Reply:
let's think about it in the future see if it's really necessary